Setup/UpgradeData.php

```diff
@@ -528,15 +528,27 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
                 }
 
                 // Insert postnl enabled data
-                $connection->insert(
+
+                $selectDeliveryActive = $connection->select()->from(
                     $table,
-                    [
-                        'scope'    => 'default',
-                        'scope_id' => 0,
-                        'path'     => 'myparcelnl_magento_postnl_settings/delivery/active',
-                        'value'    => 1
-                    ]
+                    ['config_id', 'path', 'value']
+                )->where(
+                    '`path` LIKE "myparcelnl_magento_postnl_settings/delivery/active"'
```

Comment:
LIKE zonder % gebruikt een exacte match, dus LIKe is nu niet nodig

Reply:
Done